hashring/hashring.go

```diff
@@ -187,81 +181,69 @@ func (r *HashRing) AddRemoveServers(add []string, remove []string) bool {
 	return changed
 }
 
-// hasServer returns true if the server exists in the ring, false otherwise
+// HasServer returns true if the server exists in the ring, false otherwise
```

Comment:
Suggest: "HasServer returns whether or not the specified server exists in the ring." Or at least, add a full stop.